whats-new-panel.yaml

```diff
@@ -98,3 +98,38 @@
   targeting: firefoxVersion >= 76
   trigger:
     id: whatsNewPanelOpened
+- id: WHATS_NEW_POCKET_GB
+  template: whatsnew_panel_message
+  order: 1
+  content:
+    bucket_id: WHATS_NEW_POCKET_GB
+    published_date: 1591056000000
+    title: Fascinating articles from Pocket for Firefox
+    icon_url: data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjE2IiBoZWlnaHQ9IjE5NyIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTE2Ny4yMDkgODMuMDkybC01MS4yOTYgNDguMDUzYy0yLjY0MSAzLjAwMy02Ljc5IDQuMTI5LTkuODA3IDQuMTI5LTMuNzcyIDAtNy41NDQtMS4xMjYtMTAuNTYxLTQuMTI5TDQ1LjAwMiA4My4wOTJjLTUuMjgtNS42MzItNi4wMzQtMTUuMDE3IDAtMjEuMDI0IDUuNjU4LTUuMjU1IDE1LjA4OC02LjAwNiAyMC43NDUgMGw0MC4zNTkgMzguNjY4IDQxLjExMy0zOC42NjhjNS4yOC02LjAwNiAxNC43MS01LjI1NSAxOS45OSAwIDUuMjgxIDYuMDA3IDUuMjgxIDE1LjM5MiAwIDIxLjAyNE0xOTUuOTE2Ljc4M0gyMC44NDJDOS40NzQuNzgzIDAgOS42NCAwIDIxLjE5M3Y2NS44NWMwIDU5LjY4NyA0OC41MDUgMTA5LjM2MyAxMDguMzc5IDEwOS4zNjNDMTY3Ljg3NCAxOTYuNDA2IDIxNiAxNDYuNzMgMjE2IDg3LjA0MnYtNjUuODVDMjE2IDkuNjQgMjA2LjkwNS43ODQgMTk1LjkxNi43ODQiIGZpbGw9IiNFRjQxNTYiLz48cGF0aCBmaWxsLXJ1bGU9ImV2ZW5vZGQiIGNsaXAtcnVsZT0iZXZlbm9kZCIgZD0iTTE2Ny4yMDcgODMuMDkxbC01MS4yOTcgNDguMDUzYy0yLjY0IDMuMDA0LTYuNzg5IDQuMTMtOS44MDcgNC4xMy0zLjc3MiAwLTcuNTQzLTEuMTI2LTEwLjU2LTQuMTNMNDUgODMuMDkxYy01LjI4LTUuNjMxLTYuMDM1LTE1LjAxNiAwLTIxLjAyMyA1LjY1OC01LjI1NiAxNS4wODctNi4wMDcgMjAuNzQ1IDBsNDAuMzU4IDM4LjY2OCA0MS4xMTMtMzguNjY4YzUuMjgxLTYuMDA3IDE0LjcxLTUuMjU2IDE5Ljk5MSAwIDUuMjggNi4wMDcgNS4yOCAxNS4zOTIgMCAyMS4wMjMiIGZpbGw9IiNmZmYiLz48L3N2Zz4=
+    icon_alt: Pocket for Firefox icon
+    body: Get the most thought-provoking stories from Pocket right on your new tab.
+    cta_url: home-topstories
+    cta_type: OPEN_PREFERENCES_PAGE
+    link_text: Turn on Pocket articles
+  targeting: firefoxVersion == 77 && locale == "en-GB"
+  trigger:
+    id: whatsNewPanelOpened
+- id: WHATS_NEW_BADGE_77_GB
+  content:
+    action:
+      id: show-whatsnew-button
+    badgeDescription:
+      string_id: cfr-badge-reader-label-newfeature
+    bucket_id: WHATS_NEW_BADGE_77_GB
+    delay: 300000
+    target: whats-new-menu-button
+  frequency:
+    lifetime: 100
+  priority: 5
+  targeting: firefoxVersion == 77 && locale == "en-GB" && (usesFirefoxSync || hasAccessedFxAPanel ||
```

Comment:
Just to confirm, are we only targeting 77 for this one? Because, normally, we'd do `>= 77`.